test/unit/concepts/comparison/three_way_comparable.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Christopher Di Bella.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+#include "cjdb/concepts/comparison/three_way_comparable.hpp"
+
+#include <version>
+#if defined(__cpp_lib_three_way_comparison) and __cpp_lib_three_way_comparison >= 201711
```

Comment:
Value should be 201907 (which includes P1614, which adds these concepts, and P1902 which fixes the name of the macro that paper added)

Reply:
Applied in cf7734b02373a2a2bfbfaaa3753692cb89aad6ac.